File: app/src/main/java/com/chooloo/www/callmanager/ui/contact/ContactMvpView.java
package com.chooloo.www.callmanager.ui.contact;

import com.chooloo.www.callmanager.ui.base.MvpView;

public interface ContactMvpView extends MvpView {
    void onSetup();

    void callContact();

    void smsContact();

    void editContact();

    void openContact();

    void deleteContact();

    void animateLayout();
}

File: app/src/main/java/com/chooloo/www/callmanager/ui/contact/ContactMvpView.kt
package com.chooloo.www.callmanager.ui.contact

import com.chooloo.www.callmanager.ui.base.MvpView

interface ContactMvpView : MvpView {

    // contact actions
    fun call()
    fun sms()
    fun edit()
    fun open()
    fun delete()

    fun animateLayout()
}